source/modules/task-list.js

```diff
@@ -0,0 +1,121 @@
+/**
+ * @name addTaskButton
+ * @function
+ * @description Opens or closes add-task form
+ */
+function addTaskButton() {
+    let button = document.getElementById("add-task-form");
+    if (button.classList.contains("hidden") === true) {
```

Comment:
Codacy has a fix for the issue: [Strings must use singlequote.](https://app.codacy.com/gh/cse112-sp22-group5/cse112-sp22-group5/pullRequest?prid=9407153)
```suggestion
    if (button.classList.contains('hidden') === true) {
```